jbpm-flow/src/main/java/org/jbpm/osgi/flow/core/Activator.java

```diff
@@ -42,13 +43,14 @@ public void start(BundleContext bc) throws Exception {
         this.processRuntimeReg2 = bc.registerService( new String[]{ ProcessMarshallerFactoryService.class.getName(), Service.class.getName()},
                                                                    new ProcessMarshallerFactoryServiceImpl(),
                                                                    new Hashtable() );
-        ProcessRuntimeFactory.resetInitialization();
+        RuntimeManagerFactory.Factory.reset();
+        ProcessBuilderFactory.resetInitialization();
```

Comment:
Is this change intentional - `ProcessRuntimeFactory -> ProcessBuilderFactory`? It seems to be only here, not in the master PR.